src/wasm/wasm-type.cpp

```diff
@@ -442,8 +442,8 @@ Type Type::reinterpret() const {
   }
 }
 
-FeatureSet Type::getFeatures() const {
-  auto getSingleFeatures = [](Type t) -> FeatureSet {
+FeatureSet Type::getFeatures(FeatureSet knownFeatures) const {
```

Comment:
It'd be good if we can keep this as is and don't need to supply the current features... `--enable-anyref` feature was created just to test subtypes before GC is available, so I'm not sure if we need to take that into consideration here. Would it have a problem if we just enable GC for `anyref`? cc @tlively

Reply:
This change was necessary because roundtripping a binary with `--enable-gc` would otherwise come back with `--enable-anyref` (or vice-versa), breaking the respective feature tests. But yeah, also not happy with that in the first place.

Reply:
OK this makes me wonder... Do we even need to keep `--enable-anyref` after we add `--enable-gc` flag? `--enable-anyref` was only for subtype testing, and after we add `--enable-gc`, even if GC feature is not complete, as long as it supports `anyref` type, I think we can use it for subtype testing. And all test files in test/ directory that does subtying test are named with the prefix `_all-features`, so this doesn't require changing their file names either.

Reply:
I'd be ok with essentially replacing `--enable-anyref` with `--enable-gc`. Certainly simplifies matters.